<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Futaam</class>
 <widget class="QMainWindow" name="Futaam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Futaam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>1</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>1</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Entry"/>
    <addaction name="actionDelete_Entry"/>
    <addaction name="actionEdit_Entry"/>
    <addaction name="actionSwap_Entries"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Futaam"/>
    <addaction name="actionReadme"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionStats"/>
    <addaction name="actionViewDetails"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Database</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Entry">
   <property name="text">
    <string>Add Entry</string>
   </property>
  </action>
  <action name="actionDelete_Entry">
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </action>
  <action name="actionSwap_Entries">
   <property name="text">
    <string>Swap Entries</string>
   </property>
  </action>
  <action name="actionEdit_Entry">
   <property name="text">
    <string>Edit Entry</string>
   </property>
  </action>
  <action name="actionAbout_Futaam">
   <property name="text">
    <string>About Futaam</string>
   </property>
  </action>
  <action name="actionViewDetails">
   <property name="text">
    <string>Entry Details</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="text">
    <string>Database Stats</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="text">
    <string>View Readme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
